appveyor.yml

```diff
@@ -10,28 +15,36 @@ environment:
   CHROMAPRINT_FPCALC_VERSION: 1.4.2
   DISCID_VERSION: 0.6.1
   matrix:
-  - PYTHON: C:\Python36
-    PYTHON_VERSION: 3.6.1
-    PYTHON_ARCH: 32
-  - PYTHON: C:\Python35
-    PYTHON_VERSION: 3.5.3
-    PYTHON_ARCH: 32
-build: off
-test_script:
+  - PYTHON: C:\Python35-x64
+    python_version: 3.5.3
+    python_arch: 64
+  - PYTHON: C:\Python36-x64
+    PYTHON_VERSION: 3.6.4
+    PYTHON_ARCH: 64
+build_script:
 - cmd: >-
     appveyor DownloadFile https://github.com/acoustid/chromaprint/releases/download/v%CHROMAPRINT_FPCALC_VERSION%/chromaprint-fpcalc-%CHROMAPRINT_FPCALC_VERSION%-windows-i686.zip -FileName fpcalc.zip
 
+    appveyor DownloadFile https://indy.fulgan.com/SSL/openssl-%OPENSSL_VERSION%-x64_86-win64.zip -FileName openssl.zip
+
     7z x fpcalc.zip -y
 
     copy /Y chromaprint-fpcalc-%CHROMAPRINT_FPCALC_VERSION%-windows-i686\fpcalc.exe %PYTHON%
```

Comment:
Does this provide a working fpcalc.exe anywhere that can be picked up by Picard? At least in Wine I wasn't able to configure that.

Reply:
#824 was made to fix that